python/hail/__init__.py

```diff
@@ -41,6 +42,7 @@
            'methods',
            'eval_expr',
            'eval_expr_typed',
+           'tablify',
```

Comment:
This is the wrong module. Also, I don't think this should be in the hail or hail2 package.

Reply:
I don't understand. I figured `tablify` would be generally useful to library writers.